sandbox/utils.js

```diff
@@ -37,7 +37,7 @@ function generateFormSchema(o, rootSpreadProp, accum = {}) {
         const kk = rootSpreadProp ? `${rootSpreadProp}.${k}` : k;
 
         if (o[k] !== undefined && o[k] !== null && typeof o[k] !== "function") {
-            typeof o[k] === "object" ? generateFormSchema(o[kk], kk, accum) : (accum[kk] = formMap(kk, o[k]));
+            typeof o[k] === "object" ? generateFormSchema(o[k], kk, accum) : (accum[kk] = formMap(kk, o[k]));
```

Comment:
Was this change intended?

Reply:
Yes, configurations with object depth greater than 2 will error out in this function causing the sandbox to not load. For example in my case, 
```javascript
const config = {
    node: {
        size: {
            height: 400,
            width: 500,
        }
    }
}
```
when trying to recurse into the size object (line 40) it was attempting to pass in `o[node.size]` which resulted in undefined instead of passing in simply `o[size]` since o is already the node object.

Hope my explanation makes sense!

Reply:
Cool, thanks for addressing it 👍